lib/wibox/widget/textclock.lua

```diff
@@ -1,6 +1,42 @@
 ---------------------------------------------------------------------------
 --- Text clock widget.
 --
+-- The `wibox.widget.textclock` widget is part of the Awesome WM's wiboxes
```

Comment:
The full sentence says `Awesome WM's wiboxes system` with a link to the documentation at https://awesomewm.org/apidoc/documentation/03-declarative-layout.md.html.

Still, "wiboxes system" is probably an unnecessarily complicated way to says it's a widget from Awesome library...

I'll go with `Awesome WM's widgets system` for now. The objective of this sentence was to help visualize where the class is on the internal API conception. I think this could be removed once I'll have finalized the "inheritance diagram" stuff. I guess a good diagram will be better than a strange sentence...